generic-styles/content-baked.less

```diff
@@ -314,6 +314,10 @@ h1.example-title .text {
     p:first-of-type {
       display: inline-block;
     }
+
+    .os-problem-container, .os-problem-container > p {
```

Comment:
this solution was copied from https://vendor.cnx.org/contents/zOZP3vRI@13.9:XwcQ_hAo@9/1-1-Physics-Definitions-and-Applications so we can have some confidence that it works with other variants in the content

Reply:
Yeah I did the wrong change before because I came up with the fix, when actually I should've checked the styles from `cnx` and copy them, so I fixed that based on the real thing, generic styles is copying styles from `cnx` to `rex`, so this should be fine, styles when highlighting has had issues before, depends on where you start highlighting but I think those aren't exactly related to this.